### ISSUE TITLE
nvda interrupts continuous reading

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#9644]
### Steps to reproduce:
I started reading a document in word, this is a document with 112 pages.
This document has several elements, such as figures, tables, links and other formats that I do not know in its entirety.
When you start reading this document, with nvda + a.
After a while the reading is stopped.
It is not another open window or loss of focus, nvda does not announce the opening of any window and also returns the reading by pressing down arrow.
I have not noticed yet some element found by nvda that makes him interrupt the reading.
Below is a log where I noticed several errors, nvda is without complements and using the synthesizer one core in Portuguese Brazil.
### Actual behavior:
Open the word and start reading a large, complex document.
After a while the reading is interrupted, plus returns if you press down arrow.
### Expected behavior:
nvda should be able to read the document without interruptions until the end if the user wants.
#### NVDA installed/portable/running from source:
install
#### NVDA version:
: threshold-17401,a939e3c0
#### Windows version:
10 17763.529
#### Name and version of other software in use when reproducing the issue:
Microsoft® Word 2016 MSO (16.0.4266.1001) 64 bits 

#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
[speak.txt](https://github.com/nvaccess/nvda/files/3238184/speak.txt)

**Comment by fernando-jose-silva** [comment]
I believe that I was able to capitalize on one of the excerpts from the document where nvda presents the problem.
Open the attached word document
Press nvda + the notebook layout to read the document.
See that in the last line the nvda le up to 5.
The line has more unread content by nvda with nvda + a:
5. APOIO ÀS FAMÍLIAS

**Comment by fernando-jose-silva** [comment]
[Em 2018.docx](https://github.com/nvaccess/nvda/files/3238286/Em.2018.docx)

**Comment by Brian1Gaff** [comment]
I'm assuming you have not seen this in the other snap branches of alpha and 
beta?
 Threshold is rather a special case, and I've seen lots of minor rough 
edges. I'd be worried if stable or beta did this.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
-----

**Comment by fernando-jose-silva** [comment]
no, in alpha verbs I have not seen this problem.
Follows a message in outlook also with this problem.
Open the message, press nvda + a to read.
Note that reading is interrupted in several places.
To resume reading press down arrow.
Another computer.
nvda
threshold-17401, a939e3c0
windows 10 18362.116
[SETE PRÁTICAS PARA A PAZ.msg.txt](https://github.com/nvaccess/nvda/files/3242340/SETE.PRATICAS.PARA.A.PAZ.msg.txt)

**Comment by Adriani90** [comment]
@fernando-jose-silva can you reproduce this on NVDA 2019.3.1 or on NVDA 2020 Beta?

**Comment by fernando-jose-silva** [comment]
everything works now
very different software versions
nvda alfa 19992
office 2013
windows 7

**Comment by Adriani90** [comment]
I cannot reproduce this with One Core voice on Windows 10 1909 Update and MS Office 2016 in the documents attached to this issue.
Usually, NVDA interupts reading when a toast notification from any application in the background appears. But for this behavior there is another issue and can only be solved by speech refactor.
Thanks also for your reporting Fernando.
I am closing this for now as works for me.
